install.sh

```diff
@@ -455,10 +455,17 @@ setup_selinux() {
         rpm_site="rpm-testing.rancher.io"
     fi
 
-    policy_hint="please install:
+    if awk -F'=' '/ID_LIKE/ {print $2}' /etc/os-release|grep -q "suse"; then
```

Comment:
Using `awk` here does not seem optimal to me. From what I understand, you want to get the value of `ID_LIKE` (if it exists) in `etc/os-release` and if it exists and equals `'suse'` go into the first part of the `if`-block. I would recommend letting the script source the `/etc/os-release` file instead (since it is in shell script format), and then compare the variable `ID_LIKE` (which might then be set or not set) with `'suse'`. Then you can skip `awk` and skip `grep` and get something much simpler:

```shell
. /etc/os-release
if [ "${ID_LIKE:-}" = suse ] ; then
```